CONTRIBUTING.md

```diff
@@ -237,24 +237,35 @@ request. Typically reviewers will review the code for obvious errors, as well as
 test out the patch set and opine on the technical merits of the patch. Project
 maintainers take into account the peer review when determining if there is
 consensus to merge a pull request (remember that discussions may have been
-spread out over GitHub, mailing list and IRC discussions). The following
+spread out over GitHub, mailing list and IRC discussions).
```

Comment:
Could mention here that linking to those discussions when doing review/concept ACKing is encouraged.